tests/test_grains_minicircle.py

```diff
@@ -135,7 +117,7 @@ def test_remove_small_objects(minicircle_small_objects_removed: Grains, plotting
     )
     return fig
 
-
+'''
```

Comment:
Is there a reason this test has been commented out?

If it is no longer required it can be removed (along with the associated `test/img/test_mask_overlay.png`), otherwise the quotes around can be removed and the test re-instated.

EDIT : Ah, I've read through `grains.py` now and there is no `z_range` passed in and no thresholding done to produce a `z_threshed` image so this test is no longer required.

Reply:
Yep it was changed but the image is still being used for a test elsewhere. Removed the redundant test in c328770